### DISCUSSION TITLE
Trouble removing a component of the TownStatusScreen.

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6445] [Q&A] [answered]
so its probs a stupid issue but
I overwrit the Founded by file to get rid of it but now it just says

![image](https://user-images.githubusercontent.com/116373326/213490060-8b309ae5-0aa9-4b96-bb20-233532531388.png)

What do i do?

**Comment by LlmDl** [comment, marked as answer]
You cannot get rid of it using language files, only by using the TownyAPI to remove the component from the StatusScreen can you remove it.

Listening for the TownStatusScreenEvent, you get the StatusScreen, and remove the component that has the key "founder".

**Reply by TheWeirdCoder** [reply]
How do i edit the TownyAPI?

**Reply by LlmDl** [reply]
If you can build a plugin or would like to commission a plugin I would drop into the discord and @ me. It's a lot easier to discuss things there.
